en/firefox/browsers/mobile/index.ftl

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/mobile/
+
+# HTML page title
+browsers-mobile-firefox-mobile-browsers-put = { -brand-name-firefox } mobile browsers put your privacy first
+
+# HTML page description
+browsers-mobile-overview-of-all-mobile = An overview of all { -brand-name-firefox } mobile browsers ({ -brand-name-firefox } for { -brand-name-android }, { -brand-name-firefox } for { -brand-name-ios }, { -brand-name-firefox-focus })
+browsers-mobile-at-home-or-on-the-move = At home or on the move, you deserve privacy on every device. Enhanced Tracking Protection is automatic in every { -brand-name-firefox } mobile browser.
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words; please omit the strong tags if they need to be around multiple words in your language.
+browsers-mobile-get-the-mobile-browser-privacy = Get the mobile browser that puts your <strong>privacy</strong> first
+browsers-mobile-firefox-for-android = { -brand-name-firefox } for { -brand-name-android }
+browsers-mobile-infinitely-customizable-private = Infinitely customizable, private and secure, { -brand-name-firefox } for { -brand-name-android } is a lightning-fast browser that will never sell you out.
+browsers-mobile-firefox-for-ios = { -brand-name-firefox } for { -brand-name-ios }
+
+# "Enhanced Tracking Protection" is a feature name and should be capitalized.
```

Comment:
Many languages don't have lower or upper cases. Shall we change it to `"Enhanced Tracking Protection" is a feature name and should match what's used in the product.`